CONTRIBUTING.md

```diff
@@ -1,46 +1,87 @@
-parse through style changes and code changes at the same time. _pew pew_
+Open an issue on our [issues list](https://github.com/cdcgov/prime-simplereport/issues) which describes the feature you would like to see, why you need it, and how it should work. Please try to be as descriptive as possible. We can't guarantee that we'll be able to develop a requested feature, but we will do our best to give it the care and consideration it deserves.
 
-### Client (Javascript)
+## Contributing code and documentation changes
 
-We're just defaulting to `prettier` for formatting and styling. Easiest thing to do is enable format
-on save to avoid crazy weird formatting changes in pull requests. Inside the frontend some of this
-is handled with tests.
+If you would like to contribute a new feature or a bug fix, we ask that you follow a few guidelines. We have a dedicated team of engineers and support staff that works on this project on a daily basis, and following these steps is essential to ensuring new pieces are interoperable, with no duplicated effort.
 
-### API (Java)
+* Before you get started, look for an issue you would like to work on. Please make sure this issue is not currently assigned to someone else. If there is no existing issue for your idea, please open one.
+* Once you have selected an issue, please discuss your idea for the fix or implementation in the comments for that issue. It's possible that someone else may have a plan for this that the team has discussed, or that there are particular complexities that you should know about before starting the implementation. There are often several ways to fix a problem, and it is essential to find the right approach before spending time on a PR that cannot be merged.
+* Once we've discussed your approach, and everything looks good, we will give you a formal approval to begin development. Please do not start your development work before you get that approval; we don't want you to waste your time on a particular issue if someone is already hard at work on it, or if there's something else in the way!
 
-The initial code uses the default code formatting settings from Spring Tool
-Suite, because they are there. As we develop opinions on what changes should
-be made to those settings, we will add them here.
+We use the `good first issue` label to mark the problems we think will be suitable for new contributors.
 
-In addition, we have a small set of checkstyle rules: these  rules are intended
-to remain few in number, but by the same token they should not be bypassed
-without an extremely compelling reason.
\ No newline at end of file
+### Fork and clone the repository
+
+You will need to fork the main code or documentation repository and clone it to your local machine. See
+[github help page](https://help.github.com/articles/fork-a-repo) for help. 
+
+Create a branch for your work. 
+
+If a you need to base your work on someone elses branch, talk to the branch owner and work something out.  
```

Comment:
someone else's

Reply:
Thanks for the catch! That'll go in the fix commit, too.